src/utils.ts

```diff
@@ -16,15 +24,20 @@ function requestApi (this: PureCloudWebrtcSdk, path, { method, data, version, co
   return request.send(data); // trigger request
 }
 
-function rejectErr (this: PureCloudWebrtcSdk, message: any, details: any) {
+export function rejectErr (this: PureCloudWebrtcSdk, message: string, details: any) {
   const error = new Error(message);
   error['details'] = details;
   this.emit('error', message, details);
   throw error;
 }
 
-export {
-  buildUri,
-  requestApi,
-  rejectErr
-};
+// this is duplicated in streaming-client and valve. It may need to be its own package.
```

Comment:
Maybe. If so I want it to be very very small with 0 dependencies. There's jwt-decode now, but it's got a lot of what seems like unnecessary cruft and additional subdependencies.